client/consensus/beefy/src/aux_schema.rs

```diff
@@ -63,8 +63,8 @@ where
 
 	match version {
 		None => (),
-		Some(1) | Some(2) => (), // versions 1 & 2 are obsolete and should be simply ignored
-		Some(3) => return load_decode::<_, PersistedState<B>>(backend, WORKER_STATE_KEY),
+		Some(1) | Some(2) | Some(3) => (), // versions 1, 2 & 3 are obsolete and should be ignored
+		Some(4) => return load_decode::<_, PersistedState<B>>(backend, WORKER_STATE_KEY),
```

Comment:
nit (don't have enough context so can be ignored if older versions could be loaded in the future too)

```suggestion
		Some(version) if version < CURRENT_VERSION => (), // versions 1, 2 & 3 are obsolete and should be ignored
		Some(CURRENT_VERSION) => return load_decode::<_, PersistedState<B>>(backend, WORKER_STATE_KEY),
```